b3/parsers/battleye/abstractParser.py

```diff
@@ -59,6 +59,7 @@
 #                     - correct capitalization in _use_color_codes
 # 16/04/2015 - 1.3.3  - uniform class variables (dict -> variable)
 #                     - implement missing abstract class methods
+# 01/31/2016 - 1.3.4  - Added lock and unlock commands
```

Comment:
This is going to conflict with https://github.com/BigBrotherBot/big-brother-bot/commit/c3c1d7e1d46c50c6d455acfe632ef1661f76fb23.
You should *revert* this commit and *rebase* your iourt43 branch on top of **BigBrotherBot::master**. Ultimately, you should squash all your commits together into a single one.